server/client.go

```diff
@@ -2192,6 +2205,24 @@ func (c *client) deliverMsg(sub *subscription, mh, msg []byte) bool {
 		return false
 	}
 
+	// Do a fast check here to see if we should be tracking this from a latency
+	// persepective. This will be for a request being received for an exported service.
+	// This needs to be from a non-client (otherwise tracking happens at requestor).
+	if c.kind != CLIENT && client.kind == CLIENT && len(c.pa.reply) > minReplyLen {
+		// FIXME(dlc) - We may need to optimize this.
+		if client.acc.IsExportServiceTracking(string(c.pa.subject)) {
+			// If we do not have a registered RTT queue that up now.
+			if client.rtt == 0 {
```

Comment:
That is racy.. I don't mind that we get a false positive, just saying that `-race` would complain if you had proper test (maybe have Options.PingInterval with smallish interval and you should get it).

Reply:
client.rtt? We are holding the lock.

Reply:
my bad...